x/mint/keeper/keeper.go

```diff
@@ -176,6 +185,11 @@ func (k Keeper) DistributeMintedCoin(ctx sdk.Context, mintedCoin sdk.Coin) error
 	if err != nil {
 		return err
 	}
+
+	// Take the current balance of the developer rewards pool and remove it from the supply offset
+	developerAccountBalance := k.bankKeeper.GetBalance(ctx, k.accountKeeper.GetModuleAddress(types.DeveloperVestingModuleAcctName), mintedCoin.Denom)
+	k.bankKeeper.AddSupplyOffset(ctx, mintedCoin.Denom, developerAccountBalance.Amount)
```

Comment:
Why doesn't this make the supply offset go infinite (adding the entire dev vesting balance each epoch)? It should just be adding the `devRewardCoin.Amount` to the supply offset.

Reply:
oh your adding the whole current balance, and then subtracting the new balance after distribution?

Seems overly complex versus devRewardCoin adding. Is there a rounding error for why we do this?

Reply:
Seemed like the easiest way to make sure it's always in sync with the balance of the Developer Rewards Account